code/lib/core-server/src/build-dev.ts

```diff
@@ -173,4 +175,6 @@ export async function buildDevStandalone(options: CLIOptions & LoadOptions & Bui
     managerTotalTime,
     previewTotalTime,
   });
+
+  return { port };
```

Comment:
Remove the return statement in line 161 and wrap

```
const name =
      frameworkName.split('@storybook/').length > 1
        ? frameworkName.split('@storybook/')[1]
        : frameworkName;

    outputStartupInformation({
      updateInfo: versionCheck,
      version,
      name,
      address,
      networkAddress,
      managerTotalTime,
      previewTotalTime,
    });
```

into an else statement

Reply:
Done! :D